### PR TITLE
Basic Wallet Info with Transactions

### PR DESCRIPTION
**Pull request by chris-aeviator**
- Implements a basic transaction history
- optimized on zero transactions also
- shows account name (with edit button -> `options.html`)
- shows USD amount calculated from current BTC rate 

## Transacting a lot
image
**cursor is hovering the time entry, dunno why not visible**
## No transactions yet

image

### CODE REVIEW

**Comment by chris-aeviator** [comment]
TODO:

- [ ]  remove wallet reference from connect src

**Comment by bumi** [comment]
:shipit: merging this, and we do further updates in new PRs.

File: src/extension/background-script/connectors/lnd.js [hunk 1]
```diff
@@ -61,6 +61,12 @@ class Lnd extends Base {
     });
   };
 
+  getTransactions = () => {
+    return this.request("GET", "/v1/transactions", undefined, {
```
Comment (bumi) [inline src/extension/background-script/connectors/lnd.js:65]: oh and this should be "list payments"
https://api.lightning.community/#v1-payments
https://api.lightning.community/#listpayments

The payments are the lightning payments. Transactions are the on-chain bitcoin transactions.

Comment (chris-aeviator) [reply src/extension/background-script/connectors/lnd.js:65]: thanks for making that clear - was confused already that they seemed not up to date

Comment (chris-aeviator) [reply src/extension/background-script/connectors/lnd.js:65]: https://github.com/bumi/lightning-browser-extension/issues/45

**Pull request merged**